caab-service/src/main/java/uk/gov/laa/ccms/caab/api/entity/Address.java

```diff
@@ -65,15 +69,12 @@ public class Address implements Serializable {
   private String careOf;
 
   @Column(name = "PREFERRED_ADDRESS", length = 50)
-  @Access(jakarta.persistence.AccessType.PROPERTY)
+  @Access(AccessType.PROPERTY)
```

Comment:
I think we only need to use this annotation if we want to force access via the getter/setter. And this would have been done in the PUI because they hid business logic in them!

Reply:
removed